h/templates/rss.xml.jinja2

```diff
@@ -22,7 +22,7 @@ render the feed to RSS XML.
        <title>{{ item.title }}</title>
        <description>{{ item.description|safe }}</description>
        <pubDate>{{ item.pubDate }}</pubDate>
-       <guid isPermaLink="true">{{ item.guid }}</guid>
+       <guid isPermaLink="false">{{ item.guid }}</guid>
```

Comment:
Hmm, fair enough, since these are not actually URLs then I suppose they can't really be called permalinks